### PR TITLE
build: automatically create release

### PR DESCRIPTION
**Pull request by TrySound**
Here added workflow to automatically create release when tag is pushed to the repo.

All unconfigured types will go into "Other changes" section

Example of changelogs
https://github.com/TrySound/test-auto-release/releases

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      contents: write
+    steps:
+    - uses: ardalanamini/auto-changelog@v4
+      id: changelog
+      with:
+        commit-types: |
+          feat: New Features
+          fix: Bug Fixes
+          docs: Documentation
+          experimental: Experimental features
```
Comment (istarkov) [inline .github/workflows/release.yml:21]: Should be matched ?
https://github.com/webstudio-is/webstudio/blob/3fa4afa2691227a77ded5248a6b3cdfb05c18630/.github/workflows/lint-pull-request.yaml#L29-L41

**Review by istarkov — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (TrySound) [reply .github/workflows/release.yml:21]: Most of these are not important to user and many we just don't use

Comment (istarkov) [reply .github/workflows/release.yml:21]: What it will do if chore etc will be in our PRs

Comment (TrySound) [reply .github/workflows/release.yml:21]: Put it into "Other changes" section

**Pull request merged**